### PR TITLE
Change the way to get the name of the user from auth context object 

### PR DESCRIPTION
**Pull request by basultobdKs**
Is the best place to put the function?

As per in #39

### CODE REVIEW

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -22,19 +23,20 @@ class NavBar extends React.Component {
...
-    const {
-      context: {
-        sessionInfo: {
-          user: { name }
-        },
-        onLogout
-      }
-    } = this.props;
+    const { context } = this.props;
+    const getUserName = compose(
```
Comment (mishelashala) [inline src/components/NavBar/NavBar.js]: relocate function outside render (and component)

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -4,8 +4,7 @@ import NavBarContainer from "./NavBarContainer";
 import { AvatarButton } from "./AvatarButton";
 import { UsernameLabel } from "./UsernameLabel";
 import { NavBarMenu } from "./NavBarMenu";
-import { compose, prop, defaultTo } from "lodash/fp";
-
+import { getUserName } from "../../utils/propertiesGetters";
```
Comment (mishelashala) [inline src/components/NavBar/NavBar.js]: Rename from `propertieGetters` to `sessionInfo`

Comment (basultobdKs) [reply src/components/NavBar/NavBar.js]: For `sessionInfo`? Certainly I'm not satisfied using a Utils file approach

Comment (mishelashala) [reply src/components/NavBar/NavBar.js]: Me neither, do you have any suggestion?

**Pull request merged**